src/test/java/webApp/webControllerTest.java

```diff
@@ -184,6 +225,15 @@ public void testAddAnswerErrors() throws Exception {
                 .andExpect(content().string(containsString("error")))
                 .andExpect(content().string(containsString("Value for question \\\"q3\\\" outside of range: Want 1 to 5 but got 1024")));
 
+        surveyString1 = "{ \"id\" : " + id1 + ", \"questions\" : [" +
+                "{ \"type\": \"numberQuestion\", \"question\": \"q3\", \"numberAnswer\" : -123 }]}";
+
+        this.mockMvc.perform(post("/addAnswers").contentType("application/json")
+                .content(surveyString1)).andExpect(status().isOk())
+                .andExpect(content().string(containsString("error")))
+                .andExpect(content().string(containsString("Value for question \\\"q3\\\" outside of range: Want 1 to 5 but got -123")));
```

Comment:
I think we should avoid testing this string because we don't want the test to fail if we change the error string. If anything, you can check if the string contains "outside of range" just to make sure it's generally the right error.